packages/slate-react/src/components/slate.tsx

```diff
@@ -33,6 +33,13 @@ export const Slate = (props: {
   }, [key, onChange])
 
   EDITOR_TO_ON_CHANGE.set(editor, onContextChange)
+  IS_DESTROYED.set(editor, false)
+
```

Comment:
Instead of having a separate property to track this, could we instead use the effect to clear the EDITOR_TO_ON_CHANGE property?

Reply:
i changed. haha.. i have been forgotten.